docs/_sidebar.md

```diff
@@ -13,8 +13,17 @@
 
 
 - [Available Services](/services/README.md)
+- [Terms and Abbreviations](/terms/terms.md)
+  * [Geodesy](/terms/geodesy.md)
+  * [Mapping](/terms/mapping.md)
+  * [GIS](/terms/gis.md)
+  * [Organizations and Companies](/terms/organizations_companies.md)
+  * [Standards](/terms/standards.md)
+  * [Protocols](/terms/protocols.md)
+  * [Technologies](/terms/technologies.md)
```

Comment:
I think it shouldn't be as part of the nav bar but as a table of content inside Terms and Abbreviations

The current hierarchy creates a very noisy side bar

Reply:
fixed